src/components/ipc-middle/IpcMiddlewareContainer.js

```diff
@@ -0,0 +1,29 @@
+import { connect } from 'react-redux';
+// import { fileSave } from './../../actions/menu';
+import { canvasColorChange } from './../../actions/canvas';
+import IpcMiddleware from './IpcMiddleware';
+
+const mapStateToProps = (props) => {
+    console.log(props);
```

Comment:
Try not to leave any console statements in the code, it slows things down when debugging

Reply:
Gone now!